docs/dev/basics.rst

```diff
@@ -61,5 +62,71 @@ Code formatting may sound like a topic for the pedantic, however it helps if
 the code in the project is formatted in the same way. The following rules
 apply:
 
-- use tabs instead of spaces
+- use tabs instead of spaces (set your editor to show tabs as two spaces.) Only
+  exceptions are Makefiles, that need spaces in some places (for example the
+  Package definition)
```

Comment:
This is plain wrong. Set your editor to the tab width you like (personally, I use 8 spaces).